src/components/screen/SignIn/index.tsx

```diff
@@ -190,7 +193,31 @@ function SignIn(props: Props): ReactElement {
   };
 
   const appleLogin = async (): Promise<void> => {
-    console.log('apple login');
+    setSigningInApple(true);
+    try {
+      const credential = await AppleAuthentication.signInAsync({
+        requestedScopes: [
+          AppleAuthentication.AppleAuthenticationScope.EMAIL,
+          AppleAuthentication.AppleAuthenticationScope.FULL_NAME,
+        ],
+      });
+
+      const {
+        user, // The unique identifier for the user
+        email,
+        fullName,
+      } = credential;
+
+      // @TODO Signed in using user
```

Comment:
@JeffGuKang  Todo?

Reply:
It should be updated after social login function is done on the backend. So I added a TODO tag. It can be removed if you think the tag is unnecessary.

Reply:
Removed :) the TODO tag.